src/main/java/org/jenkinsci/plugins/workflow/actions/ErrorAction.java

```diff
@@ -140,6 +147,20 @@ public String getUrlName() {
         return candidate;
     }
 
+    private static @CheckForNull String findId(Throwable error, Set<Throwable> visited) {
+        if (error == null || !visited.add(error)) {
```

Comment:
Not sure if we actually need the `visited` set here. Maybe it only matters in pathological cases?